CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## [v1.0.49](https://github.com/stargate/stargate/tree/v1.0.49) (2022-02-07)
+
+[Full Changelog](https://github.com/stargate/stargate/compare/v2.0.0-ALPHA-1...v1.0.49)
```

Comment:
I'm not sure how we're handling v2 vs v1, but should this be all the change from `v1.0.48...v1.0.49` instead of `v2.0.0-ALPHA-1...v1.0.49`?

Reply:
Hmmh. Good question. Probably?

Reply:
Fixed. Thank you for catching this @mpenick !